packages/api-client/src/granules.ts

```diff
@@ -243,29 +243,40 @@ export const applyWorkflow = async (params: {
  * Delete a granule from Cumulus via the API lambda
  * DELETE /granules/${granuleId}
  *
- * @param {Object} params             - params
- * @param {string} params.prefix      - the prefix configured for the stack
- * @param {string} params.granuleId   - a granule ID
- * @param {Function} params.callback  - async function to invoke the api lambda
- *                                      that takes a prefix / user payload.  Defaults
- *                                      to cumulusApiClient.invokeApifunction to invoke the
- *                                      api lambda
- * @returns {Promise<Object>}         - the delete confirmation from the API
+ * @param {Object} params                      - params
+ * @param {pRetry.Options} params.pRetryObject - pRetry options object
+ * @param {number} params.expectedStatusCode   - expected API status code
+ * @param {string} params.prefix               - the prefix configured for the stack
+ * @param {string} params.granuleId            - a granule ID
+ * @param {Function} params.callback           - async function to invoke the api lambda
+ *                                               that takes a prefix / user payload.  Defaults
+ *                                               to cumulusApiClient.invokeApifunction to invoke the
+ *                                               api lambda
+ * @returns {Promise<Object>}                  - the delete confirmation from the API
  */
 export const deleteGranule = async (params: {
   prefix: string,
   granuleId: GranuleId,
+  expectedStatusCode?: number,
```

Comment:
I thought we were deliberately avoiding this type of change in favor of always doing a try/catch? The argument being that helpers in this package should always have a consistent behavior of throwing errors when they are encountered and it shouldn't be configurable based on toggle parameters

Reply:
Right you are.    I'll update this.

Reply:
c1272e171 looks better to me.     Not sure what I was thinking on that one.